### ISSUE TITLE
Translate form labels

### ISSUE THREAD

**Issue by anormand** [unimelb/The-University-of-Melbourne-Web-Templates#11]
### Problem:

The translate form does not have any labels associated with it.
### WCAG Guideline

Success Criteria 3.3.2 - Labels or Instructions

Sufficient labels, cues, and instructions for required interactive elements are provided.
### Remedy:

Add a label to the form.
### Example

Change
`<strong>Translate this page</strong>
...
<select name="tl">`

to
`<label for="tl"><strong>Translate this page</strong></label>
...
<select id="tl" name="tl">`
